### ISSUE TITLE
sug: consider rename .txt files to .md

### ISSUE THREAD

**Issue by davvid** [pvlpenev/fullstacklisp#6]
If you rename e.g. `chapter1.txt` to `chapter1.md` then github will properly render them as markdown. Just a sug. Great work!

**Comment by azzamsa** [comment]
Yes, I also wondering why Pavel Penev use .txt instead of .md , 

- Is he use different script to convert to PDF ?
- or it is Leanpub convention ?

As far I use, I use .org , .tex and .md  , and convert the last one using pandoc.
